app/src/gecko/java/org/mozilla/focus/web/GeckoViewPrompt.java

```diff
@@ -573,7 +594,8 @@ private static void setCalendarTime(final Calendar cal, final TimePicker picker)
         }
     }
 
-    public void promptForDateTime(final GeckoSession session, final String title,
+    @Override
+    public void onDateTimePrompt(final GeckoSession session, final String title,
```

Comment:
\[PMD\] INFO: The method 'onDateTimePrompt' has a Standard Cyclomatic Complexity of 23\.